QueryGenerator.php

```diff
@@ -449,6 +449,48 @@ private function getGlue($method) {
          return ") OR (";
       }
    }
+
+   /**
+    * Debug method to print out the full query with the params inserted at the correct locations
+    */
+   public function debugStringifyQuery(): string {
```

Comment:
Not a big deal, but I'd expect a name more like `toDebugString()`, following the pattern of the magic `__toString()` method. I think it's a good idea to put `debug` in there because we really don't want to encourage using this for anything but debugging.

Reply:
Yep, thats why I opted not to override toString(). Good idea!

Reply:
Updated in [69e09e4](https://github.com/iFixit/PopSQL/pull/14/commits/69e09e40c88993aba6a9b60c28023fff72dcd85e)